stable/database/values.yaml

```diff
@@ -152,8 +156,13 @@ database:
     # labels
     # Additional Labels given to the SMs started
     labels: {}
+
     engineOptions: {}
 
+    # named key/value pairs that need to be passed to the image, such as
+    # keystore: "/etc/nuodb/keys/nuoadmin.p12"
```

Comment:
Isn't this hardcoded in the secret mounting? What happens if the user configures something like this:
```
database.te.otherOptions.keystore: "/etc/nuodb/keys/stanimir.p12"
database.sm.otherOptions.keystore: "/etc/nuodb/keys/stanimir.p12"
admin.tlsKeyStore.key:  "stanimir.p12",
```
Is it going to work?

Reply:
yeah, that should work too. Keep in mind that all these commented out sections only exist as documentation/examples.